static/src/javascripts/projects/common/modules/ui/dropdowns.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import bean from 'bean';
+import fastdom from 'fastdom';
+
+const containerSelector = '.dropdown';
+const buttonCN = 'dropdown__button';
+const contentCN = 'dropdown__content';
+
+const updateAria = (container: Element): void => {
+    const v: boolean = container.classList.contains('dropdown--active');
+    const content = [...container.getElementsByClassName(contentCN)];
+    const button = [...container.getElementsByClassName(buttonCN)];
```

Comment:
Here's an example of what I was referring to in my initial comment: https://jsbin.com/qomevaxosi/edit?html,js,console

Spread unpacks the values from the array and the node list into a single array.